data/modules/Taxi/Taxi.lua

```diff
@@ -179,31 +182,37 @@ local onChat = function (form, ref, option)
 		form:SetMessage(howmany)
 
 	elseif option == 3 then
-		if not Game.player.cabin_cap or Game.player.cabin_cap < ad.group then
+		if Passengers.CountFreeCabins(Game.player) < ad.group then
```

Comment:
BUG: this counts each cabin as a single berth; need to count total number of free berths.


This potentially has some nuance - a group may be willing to be split up amongst multiple cabins, but may not be willing to share a cabin with strangers. Ditto with VIP or secretive people.

This is out-of-scope for this PR, but should probably be captured somewhere. A group of people (which could consist of just a single person) should have a group-id, and an indicator whether they are willing to share cabins with non-group passengers.

Reply:
Yeah, passenger gameplay is tentatively planned to see further changes involving quality of accomodation and a passenger management UI. For this specifically, I'd recommend writing a proposal / stub page on passenger gameplay and submitting it to the Dev Docs as a PR - likely no review required :)

Reply:
https://github.com/pioneerspacesim/pioneer-developer-docs/pull/1

Reply:
`Passengers.CountFreeCabins()` returns the number of unoccupied passenger berths, not passenger cabins. I've renamed it to `Passengers.CountFreeBerths()` instead to avoid the semantic overloading of the term "cabin".